dot/sync/tip_syncer_integeration_test.go

```diff
@@ -259,12 +267,19 @@ func TestTipSyncer_handleTick_case3(t *testing.T) {
 			targetNumber: uintPtr(fin.Number),
 			direction:    network.Descending,
 			requestData:  bootstrapRequestData,
-			pendingBlock: s.pendingBlocks.getBlock(header.Hash()),
+			pendingBlock: &pendingBlock{
+				hash:    header.Hash(),
+				number:  300,
+				header:  header,
+				body:    &types.Body{},
+				clearAt: time.Time{},
+			},
 		},
 	}
 
 	w, err = s.handleTick()
 	require.NoError(t, err)
+	w[0].pendingBlock.clearAt = time.Time{}
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)

Reply:
Added `assert.Greater(t, w[0].pendingBlock.clearAt, time.Now())` to confirm clearAt is in the future.